lldb/source/API/SBSaveCoreOptions.cpp

```diff
@@ -7,6 +7,9 @@
 //===----------------------------------------------------------------------===//
 
 #include "lldb/API/SBSaveCoreOptions.h"
+#include "lldb/API/SBError.h"
+#include "lldb/API/SBFileSpec.h"
```

Comment:
SBFileSpec is unused right? Shouldn't need this include.

Reply:
Good call, I'll try to trim all the unneeded usings now that the header isn't in private interfaces